models/db.ts

```diff
@@ -0,0 +1,20 @@
+import mongoose from "mongoose";
+
+mongoose.set("strictQuery", true);
+
+const dbURI: string =
```

Comment:
Could you add the connection string with this variable:

-- .env.demo
CONNECTION_STRING="mongodb+srv://<username>:<password>@cluster0.jsllulh.mongodb.net/<database>?retryWrites=true&w=majority"